README.md

```diff
@@ -8,3 +8,7 @@ This repository is responsible for most content visible on the website [TechWork
 2. Start a local server: `bundle exec jekyll serve`
 
 Open a browser to localhost:8080
+
+## Add your city
+
+If your city's local is not mentioned in the website, link to it on the homepage [here](_layouts/home.html) and if you want a markdown page see the other examples [here](chapter_pages)
```

Comment:
Did you mean `city_local` instead of `chapter_pages`?

Reply:
I was hesitant to use the term local internal, because of its ambiguous meaning in web development context. Given that it’s not visible anywhere in the website itself I preferred chapter so as not to introduce any bugs inadvertently

Reply:
Oh, is this not linking to the `city_local/` directory you just created? (https://github.com/techworkersber/twc-site/tree/add-berlin/city_local)

Reply:
You're right! Good catch!